tests/integration/conftest.py

```diff
@@ -29,7 +31,7 @@ async def model_fixture(ops_test: OpsTest) -> Model:
 @pytest_asyncio.fixture(scope="module", name="charm")
 async def charm_fixture(pytestconfig: pytest.Config) -> str:
     """Get value from parameter charm-file."""
-    charm = pytestconfig.getoption("--charm-file")
+    charm = typing.cast(str, pytestconfig.getoption("--charm-file"))
```

Comment:
What is the reason for needing this cast please?

Reply:
I can't remember why I needed to add this, so I'll remove it for now.